tests/e2e/lib.sh

```diff
@@ -219,15 +219,18 @@ patch_resources_for_test() {
 
     kubectl -n stackrox patch svc central-loadbalancer --patch "$(cat "$TEST_ROOT"/tests/e2e/yaml/endpoints-test-lb-patch.yaml)"
     kubectl -n stackrox apply -f "$TEST_ROOT/tests/e2e/yaml/endpoints-test-netpol.yaml"
-    # shellcheck disable=SC2034
-    for i in $(seq 1 20); do
-        if curl -sk --fail "https://${API_HOSTNAME}:8446/v1/metadata" &>/dev/null; then
-            return
-        fi
-        sleep 1
+
+   for target_port in 8444 8446; do
```

Comment:
Maybe mention here what these two are for and it's sufficient for just one to be available?

Reply:
It was done by mistake. Thanks for highlighting!